src/raiden_libs/contract_info.py

```diff
@@ -19,18 +19,13 @@ def get_deployment_infos(
     contracts_version: str = None,
 ) -> Dict[str, Any]:
     try:
-        core_contract_data = get_contracts_deployed(
+        contract_data = get_contracts_deployment_info(
             chain_id=chain_id,
             version=contracts_version,
         )
-        service_contract_data = get_contracts_deployed(
-            chain_id=chain_id,
-            version=contracts_version,
-            services=True,
-        )
-        token_network_registry_info = core_contract_data['contracts'][CONTRACT_TOKEN_NETWORK_REGISTRY]  # noqa
-        monitor_contract_info = service_contract_data['contracts'][CONTRACT_MONITORING_SERVICE]
-        user_deposit_contract_info = service_contract_data['contracts'][CONTRACT_USER_DEPOSIT]
+        token_network_registry_info = contract_data['contracts'][CONTRACT_TOKEN_NETWORK_REGISTRY]  # noqa
```

Comment:
You can remove the `# noqa`  here after the change.

Reply:
That's wonderful.